packages/components/pie-assistive-text/src/index.ts

```diff
@@ -13,8 +19,36 @@ const componentSelector = 'pie-assistive-text';
  * @tagname pie-assistive-text
  */
 export class PieAssistiveText extends LitElement implements AssistiveTextProps {
+    @property()
+    @validPropertyValues(componentSelector, variants, 'default')
+    public variant?: AssistiveTextProps['variant'] = 'default';
+
+    /**
+ * Renders the assistive-text icon content.
+ * @private
+ */
+    private renderContent (): TemplateResult {
+        const { variant } = this;
+        return html`
+            ${variant === 'success' ? html`<icon-check-circle size="s" />` : nothing}
```

Comment:
out of interest can we apply classes to these icon elements? i.e. `.c-assistiveText-icon`

If so, this might be nicer for styling instead of relying on the internal class used inside the icon itself!